debian/include/full

```diff
@@ -1,7 +1,7 @@
 include(global_macros)
 include(minimal)
 
-define(`CLANG_TOOLING_VERSION', `8')
+ifelse(DEBIANVERSION, `buster', `define(`CLANG_TOOLING_VERSION', `8')', `define(`CLANG_TOOLING_VERSION', `10')')
```

Comment:
I've fixed missing packages in master, maybe rebase this PR instead?

Reply:
I reset this branch to master and started again, only including the relevant changes.